manifests/mosquitto/deployment.yaml

```diff
@@ -19,7 +23,7 @@ spec:
             - name: SOLUNAR_WS_LOG_LEVEL
               value: "1"
           name: mosquitto
-          image: docker.io/eclipse-mosquitto:1.6 
+          image: docker.io/eclipse-mosquitto@sha256:4ee2d823259ed813fc3761ac63dcf5a940231017bfb6aa4fa74615f63f1a477f
```

Comment:
I'm afraid of this. It's a major version bump. Will it still work? Same config files?
Maybe try it on the local docker-compose stack first?

Reply:
I think docker composer is using v2 for quite some time

Agreed, let's test more